include/circt/Dialect/FIRRTL/FIRRTLCanonicalization.td

```diff
@@ -125,34 +125,34 @@ def GetWidthAsIntAttr : NativeCodeCall<
 
 // node(x,name),!dnt -> node(x, "")
 def DropNameNode : Pat<
-  (NodeOp $x, $name, $annotations, $inner_sym),
-  (NodeOp $x, (GetEmptyString), $annotations, $inner_sym),
+  (NodeOp $x, $name, $annotations, $inner_sym, $hasDroppableName),
+  (NodeOp $x, (GetEmptyString),  $annotations, $inner_sym, $hasDroppableName),
```

Comment:
For these "DropName" transformations (didn't even know they existed!), should we be setting hasDroppableName to `true` rather than carrying over what it was previously? It seems like we could potentially have an empty name string with hasDroppableName=true.